conf/modules.config

```diff
@@ -471,6 +471,7 @@ process {
     }
 
     withName: KRAKENTOOLS_COMBINEKREPORTS_CENTRIFUGE {
+        errorStrategy = { task.exitStatus in [255,1] ? 'ignore' : 'retry' }
```

Comment:
Wasn't this part of the other PR?

Reply:
Yes, I had to add it here to allow local testing of this PR while waiting for the other to merge, so it'll 'disappear' once the centrifuge one is resolved.